src/features/rewards/welcomePage/index.tsx

```diff
@@ -83,11 +83,11 @@ class WelcomePage extends React.PureComponent<Props, {}> {
           <StyledBatLogo>
             <BatColorIcon />
           </StyledBatLogo>
-          <StyledH1>
+          <StyledRewardsTitle level={1}>
```

Comment:
Should be level 2. Because with level 1 we change it from 38px to 48px. Let's use level 2 which is 40px so that jump is not that huge

Reply:
Sounds good